src/Behat/Behat/Output/ServiceContainer/Formatter/JUnitFormatterFactory.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Output\ServiceContainer\Formatter;
+
+use Behat\Testwork\Output\ServiceContainer\OutputExtension;
+use Behat\Testwork\Exception\ServiceContainer\ExceptionExtension;
+use Behat\Testwork\Output\ServiceContainer\Formatter\FormatterFactory;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Behat junit formatter factory.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class JUnitFormatterFactory implements FormatterFactory
```

Comment:
I don`t understand the intention of making this final, can you explain a little in detail why this one should be final if it is an option to remove the final in future, in my understanding this is exactly the argument to not make this class final.